.github/workflows/publish-to-odr.yml

```diff
@@ -61,13 +61,13 @@ jobs:
         id: modified-files
         run: |
           # AM = Include: Added, Modified
-          echo "modified_files=$(git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.event.after }})" >> $GITHUB_OUTPUT
+
+          MODIFIED_FILEs=$(git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.event.after }})
```

Comment:
This isn't going to work - `MODIFIED_FILEs` != `MODIFIED_FILES`.

Reply:
:eyes: good spot!